syntax = "proto3";

package bakdata.lobby.v1;

message Lobby {
  string id = 1;
  string companyName = 2;
  int32 lobbyEmployeeCountFrom = 3;
  int32 lobbyEmployeeCountTo = 4;
  repeated string lobbyEmployyeNames = 5;
  string reasonForLobbying = 6;
  repeated string memberships = 7;
  repeated string donators = 8;
  int32 lobbyMoneySpentFrom = 9;
  int32 lobbyMoneySpentTo = 10;
  string lobbyMoneySpentYear = 11;
  string lobbyMoneySpentReason = 12;
}
